beeline-core/src/main/java/io/honeycomb/beeline/tracing/propagation/HttpServerPropagator.java

```diff
@@ -10,25 +11,31 @@
 import static io.honeycomb.beeline.tracing.utils.TraceFieldConstants.*;
 
 /**
- * Provides straightforward instrumentation of an an HTTP server, while also adding a
+ * Provides straightforward instrumentation of an an HTTP server, while also
+ * adding a
  * standardized set of span fields.
  *
- * <p>Example instrumentation for synchronous server code:
- * <pre>{@code
- * HttpServerRequestAdapter adaptedHttpRequest = new HttpServerRequestAdapter(actualHttpRequest);
- * HttpServerResponseAdapter adaptedHttpResponse = null;
- * Throwable error = null;
- * Span span = serverPropagator.startPropagation(adaptedHttpRequest);
- * try {
- *   actualHttpResponse = doServerWork(actualHttpRequest);
- *   adaptedHttpResponse = new HttpServerResponseAdapter(actualHttpResponse);
- * } catch (Exception ex) {
- *   error = ex;
- *   throw ex;
- * } finally {
- *   serverPropagator.endPropagation(adaptedHttpResponse, error, span);
+ * <p>
+ * Example instrumentation for synchronous server code:
+ *
+ * <pre>
+ * {
+ *     &#64;code
```

Comment:
```suggestion
 *     @code
```

Looks like an `@` got encoded?